tracing-tracy/Cargo.toml

```diff
@@ -33,7 +33,7 @@ criterion = "0.5"
 [features]
 # Refer to FEATURES.mkd for documentation on features.
 default = [ "enable", "system-tracing", "context-switch-tracing", "sampling", "code-transfer",
-            "broadcast", "callstack-inlines" ]
+            "broadcast", "callstack-inlines", "demangle", "demangle-impl" ]
```

Comment:
Are these enabled by default by the downstream tracy library? Typically I only put features that require disabling from their default enabled state in the upstream library as enabled by default. Would be good to maintain that property here.

Reply:
I don't believe so, but also the default demangling function is for C++ symbols only, so I wouldn't expect it to be. Without this most symbols remain mangled in the sampled call stacks